rest-service/manager_rest/storage/resource_models.py

```diff
@@ -2038,6 +2046,43 @@ def set_node(self, node):
     def allowed_filter_attrs(cls):
         return ['id']
 
+    @property
+    def is_status_check_ok(self):
+        """Has the last status check for this NI succeeded?
+
+        This examines the result of the most recent check_status call
+        on this node instance, and returns whether the call succeeded.
+
+        If the result is missing, the result is succeeded.
+        """
+        props = self.system_properties or {}
+        status = props.get('status') or {}
+        return bool(status.get('ok', True))
```

Comment:
This might be one of the "just 'cause" ones, but why is the bool conversion needed here? Isn't the props['status'] either bool or nothing?
I mean, if it's "false" then this'll interpret it as true which seems wrong.

Reply:
it's not needed, but it is just in case indeed. Meh, I'll keep it.